xrdp/xrdp.c

```diff
@@ -149,9 +78,9 @@ g_xrdp_sync(long (*sync_func)(long param1, long param2), long sync_param1,
 static void
 xrdp_shutdown(int sig)
 {
-    if (!g_is_wait_obj_set(g_term_event))
+    if (!g_is_wait_obj_set(g_get_term()))
```

Comment:
That's true iff (if and only if) _WIN32 is not defined. Because we're no longer supporting WIN32, we have to change this behavior to error out because without it there MIGHT be a side effect.

Updated the pull request.